securesignals-ima-dev-app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+
+    <application
+        android:name=".IMADevApplication"
+        android:label="@string/app_name"
+        android:allowBackup="true"
+        android:icon="@drawable/ic_launcher"
+        >
+        <activity
+            android:name=".MainActivity"
+            android:configChanges="keyboard|keyboardHidden|orientation|screenLayout|uiMode|screenSize|smallestScreenSize"
+            android:exported="true"
+            android:theme="@style/Theme.AppCompat.Light">
```

Comment:
Nit: I don't think you actually need a theme here.

Reply:
Android disagrees:

<img width="1949" alt="Screenshot 2023-04-19 at 10 39 18 AM" src="https://user-images.githubusercontent.com/989928/233127970-9bdf1271-3504-4147-ac7b-43be87f78f23.png">